app/views/comments/show.html.haml

```diff
@@ -1,4 +1,12 @@
 = content_for :title, @comment.post.subject
+- content_for :opengraph do
+  <meta property="og:image" content="#{avatar_uri(@comment.post.author, 150)}"/>
+  <meta property="og:image:user_generated" content="true" />
+  <meta property="og:title" content="#{@comment.post.subject}" />
+  <meta property="og:description" content="#{ strip_tags(@comment.post.body).split(' ')[0..20].join(' ') }" />
```

Comment:
I did think about it at the time - users putting in " are probably going to have a bad time of it; unfortunately. I'll probably go swap to the ```tag()``` method to clean this up in a bit.